forum/qa-plugin/report-reason/q2apro-flag-reasons-page.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+class q2apro_flag_reasons_page extends q2apro_flag_reasons_validation {
+
+    private $directory;
+    private $urlToRoot;
+
+    protected $CUSTOM_REPORT_REASON_ID = 0;
+    protected $_reportReasonRequestJSON = null;
+
+    public function __construct() {
```

Comment:
sypią się trochę klamry, o tym chyba mówiliśmy, ale tu nawet w ramach jednego plugina część kodu zapisana jest po staremu jak Q2A, część po naszemu jak PSR-2/12, część teraz jeszcze inaczej... spróbowałbym ujednolicić na PSR-2/12

Reply:
Racja. Część kodu PHP pochodzi z bazowego brancha i tam zdaje się, że klamry są od nowych linijek. W kodzie, który pisałem od nowa lub refaktorowałem, klamry są w tej samej linii. Jak rozumiem lepiej jest umieszczać klamry w nowych liniach (i to dotyczy klamer każdego bloku: funkcja, if, switch, pętla)?

Reply:
Planowaliśmy działać wg standardu PSR-2, więc jeśli chcesz iść za tym, to tu w przykładach jest dokładnie pokazane co jak się zapisuje: https://www.php-fig.org/psr/psr-2/ if, switch czy pętla mają w tej samej linii, ale już klasa czy metoda w nowej